pandas/tests/series/indexing/test_setitem.py

```diff
@@ -210,6 +210,14 @@ def test_setitem_nan_casts(self):
         ser[:5] = np.nan
         tm.assert_series_equal(ser, expected)
 
+    @pytest.mark.parametrize("dtype", ["int64", "Int64"])
+    def test_setitem_assigning_different_dtype(self, dtype):
```

Comment:
is this a test for `Series.__setitem__` or `loc.__setitem__`?

Reply:
``loc.__setitem__``. What is the appropriate file for this? ``test_indexing`` in the same folder?

Reply:
tests.indexing.test_loc

this file should only be for `Series.__getitem__`

Reply:
Done, thx